### PR TITLE
hideous sorting

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
we may not end up needed this, but pulling the nulls to the front may still be nice 🤔 

Blocker: `ailetadate` is going to be added as a sort for `v1/agendaItems` will update this after

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
Not ready for review, just sharing the ugly

File: talentmap_api/common/common_helpers.py [hunk 1]
```diff
@@ -415,6 +415,27 @@ def get_avatar_url(email):
...
+    '''
+    #   pull out the nulls
+    nullLegs = []
+    for idx, val in enumerate(agendaLegs):
+        if val['ailetadate'] is 'null':
+            nullLegs.append(val)
+            agendaLegs.pop(idx)
+
+    # sort legs
+    sortedLegs = sorted(agendaLegs, key=lambda d: d['ailetadate'])
```
Comment (mjoyce91) [inline talentmap_api/common/common_helpers.py:431]: Same here

File: talentmap_api/common/common_helpers.py [hunk 2]
```diff
@@ -415,6 +415,27 @@ def get_avatar_url(email):
...
+
+    # sort nulls by ted
+    sortedNulls = sorted(nullLegs, key=lambda d: d['ailetdtedsepdate'])
```
Comment (mjoyce91) [inline talentmap_api/common/common_helpers.py:434]: And probably want `ted` here

File: talentmap_api/common/common_helpers.py [hunk 3]
```diff
@@ -415,6 +415,27 @@ def get_avatar_url(email):
...
+
+    # stick sortednulls in the front of legs
+    return sortedNulls.extend(sortedLegs)
```
Comment (mjoyce91) [inline talentmap_api/common/common_helpers.py:437]: `extend` doesn't return anything, it simply performs the action:

Screen Shot 2022-02-15 at 1 29 30 PM

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**